controller/tests/controller_server/csi_controller_server_test.py

```diff
@@ -106,37 +113,33 @@ def setUp(self, connect):
     def test_create_snapshot_with_empty_name(self, a_enter):
         self._test_create_object_with_empty_name(a_enter)
 
-    def _prepare_create_snapshot_mocks(self, storage_agent, array_type):
+    def _prepare_create_snapshot_mocks(self, storage_agent):
         storage_agent.return_value = self.storage_agent
 
         self.mediator.create_snapshot = Mock()
         self.mediator.create_snapshot.return_value = utils.get_mock_mediator_response_snapshot(10, "snap", "wwn",
                                                                                                "snap_vol", "xiv")
         self.mediator.get_volume_name = Mock()
         self.mediator.get_volume_name.return_value = snap_vol_name
-        array_type.return_value = "a9k"
 
-    @patch("controller.controller_server.csi_controller_server.detect_array_type")
     @patch("controller.controller_server.csi_controller_server.get_agent")

```

Comment:
why remove `detect_array_type` but leave `get_agent`?

Reply:
I wanted to keep this PR diff minimal to begin the unused-code enforcement as soon as possible

Reply:
got you